packages/apps/halo-app/package.json

```diff
@@ -20,13 +20,14 @@
     "@dxos/react-appkit": "workspace:*",
     "@dxos/react-async": "workspace:*",
     "@dxos/react-client": "workspace:*",
-    "@dxos/react-ui": "workspace:*",
```

Comment:
If halo isn't using react-ui, what is?

Reply:
Nothing is right now, since react-ui is currently empty. That won't be the case for long, however.